src/panel/PanelManager.jsx

```diff
@@ -214,10 +216,25 @@ export default class PanelManager extends React.Component {
   render() {
     const { ActivePanel, options, panelSize } = this.state;
 
-    return <PanelContext.Provider value={{ size: panelSize, setSize: this.setPanelSize }} >
-      <div className="panel_container">
-        <ActivePanel {...options} />
-      </div>
-    </PanelContext.Provider>;
+    return <div>
+      <Suggest
+        inputNode={this.props.searchBarInputNode}
+        outputNode={document.querySelector('.search_form__result')}
+        withCategories
+        onChange={query => {
+          if (ActivePanel === ServicePanel) {
+            togglePanelVisibility(query.length === 0);
```

Comment:
Now that everything is managed in this component with access to all elements, we should be able to get rid of this fragile hiding mechanism and rely on a state value to conditionnaly render the Panel.